### PR TITLE
Fix DAI & USDT bridge addresses

### PR DESCRIPTION
**Pull request by AlexBHarley**
**Description**

The token list generation script specifies, by way of a fallback, the `bridgeBaseAddress` regardless of whether or not this token is present on Base. This led to a few abnormalities

- The Base token bridge address for Dai on mainnet incorrectly points to the PGN bridge contract 
- The PGN token bridge address for Dai on mainnet is missing
- The PGN token bridge address for USDT on mainnet is missing, marked as `baseBridgeAddress`

**Tests**

- Previously running the `validate` script for both USDT and DAI would fail, it now passes but this is unrelated to fixes specified above.

**Additional context**

I also took a moment to,

- Regenerate the token list
- Add config for the Base Sepolia network

**Metadata**

N/A

### CODE REVIEW

File: src/generate.ts [hunk 1]
```diff
@@ -80,6 +80,25 @@ export const generate = (datadir: string) => {
...
+ */
+const getTokenBridgeKey = (chain: string) => {
+  if (optimism.includes(chain)) {
+    return 'optimismBridgeAddress'
+  }
+  if (base.includes(chain)) {
+    return 'baseBridgeAddress'
+  }
+
+  return 'pgnBridgeAddress'
```
Comment (roninjin10) [inline src/generate.ts:99]:
```suggestion
  if (pgn.includes(chain)) {
  return 'pgnBridgeAddress'
  }
  throw new Error('unknown network ' + chain)
```
Prefer throwing to having a fallback. If we need a fall back we should fall back to optimismBaseAddress

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -1296,7 +1297,7 @@
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/data/DAI/logo.svg",
       "extensions": {
-        "baseBridgeAddress": "0x4200000000000000000000000000000000000010",
+        "pgnBridgeAddress": "0x4200000000000000000000000000000000000010",
```
Comment (roninjin10) [inline optimism.tokenlist.json:1300]: THis change makes it so we will have to bump the version number in package.json a major version

**Review by roninjin10 — DISMISSED** [review]

File: optimism.tokenlist.json (hunk 2 above)
Comment (AlexBHarley) [reply optimism.tokenlist.json:1300]: I guess so yes

File: src/generate.ts [hunk 3]
```diff
@@ -80,6 +80,27 @@ export const generate = (datadir: string) => {
     )
 }
 
+const optimism = ['optimism', 'optimism-goerli', 'optimism-sepolia']
+const base = ['base', 'base-goerli', 'base-sepolia']
+const pgn = ['pgn', 'pgn-sepolia']
+
+/**
+ * TODO: work out a better way to key these, perhaps by ID because
+ * the fallback to pgn isn't great for other Superchain networks
```
Comment (roninjin10) [inline src/generate.ts:89]:
```suggestion
 * TODO: work out a better way to key these, perhaps by ID
```

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Looks good bump package.json to 10.0.0 and then this is GTG

**Comment by AlexBHarley** [comment]
@roninjin10 this seems good from my side now

**Comment by AlexBHarley** [comment]
@roninjin10 this keeps getting out of date, would you mind forking and keeping up to date if this is not going to be merged anytime soon?

**Review by roninjin10 — DISMISSED** [review]
One last issue is that a snapshot test is failing. Running `npm run test -u` should fix it

**Review by roninjin10 — DISMISSED** [review]
Looks good we will need to rebase it though because of a merge conflict

**Comment by annieke** [comment]
@AlexBHarley can you rebase this PR? thanks so much!

**Comment by AlexBHarley** [comment]
@annieke @roninjin10 it looks like this PR is now not needed. The way keys (ie `{chainName}BridgeAddress`) are generated has changed and as a by product the DAI + USDT issues I flagged have been resolved.

**Pull request closed without merging**